static/js/attachments_ui.js

```diff
@@ -103,6 +103,21 @@ function render_attachments_ui() {
                 return item.name.toLocaleLowerCase().includes(value);
             },
             onupdate() {
+                const $uploaded_files_table_search = $search_input.val();
+
+                if (
+                    uploaded_files_table_widget.get_current_list().length === 0 &&
+                    $uploaded_files_table_search !== ""
+                ) {
+                    $(".uploaded_files_table").addClass("empty-table");
+                    $uploaded_files_table.attr("data-empty", "No files match your current filter.");
+                } else if (uploaded_files_table_widget.get_current_list().length === 0) {
+                    $(".uploaded_files_table").addClass("empty-table");
+                    $uploaded_files_table.attr("data-empty", "You have not uploaded any files.");
+                } else {
+                    $(".uploaded_files_table").removeClass("empty-table");
+                    $uploaded_files_table.attr("data-empty", "You have not uploaded any files.");
+                }
```

Comment:
This code is duplicated multiple times for different tables. Would be better if we can use a common function with appropriate parameters for this.

Reply:
@sahil839 I have updated the PR you can re-review.